### DISCUSSION TITLE
tabs 更多按钮的下拉框在qiankun子应用中显示异常

### DISCUSSION THREAD

**Discussion by luckjoke** [ant-design/ant-design#38424] [Q&A]
tabs 更多按钮的下拉框在qiankun子应用中显示不出来，应该也是挂载到body的问题，但select这种可以配置popupContainer，tabs要怎么配置，renderTabBar可以实现自定义的下拉框吗？renderTabBar给的示例有点简单，看不出来怎么拆开。

**Comment by zombieJ** [comment]
@kuitos，有推荐配置么？

**Reply by kuitos** [reply]
就是需要有一个类似 popupContainer的配置，全局的 ConfigProvider 支持配置tabs吗？

**Reply by yuantongkang** [reply]
上游 rc-tabs 已经支持了
